pkgs/development/tools/database/sqlitebrowser/default.nix

```diff
@@ -1,32 +1,45 @@
-{ mkDerivation, lib, fetchFromGitHub, qtbase, qttools, sqlite, cmake }:
+{ mkDerivation, lib, fetchFromGitHub, cmake, antlr
+, qtbase, qttools, qscintilla, sqlite }:
 
-mkDerivation rec {
-  version = "3.10.0";
+let
+  qscintillaLib = (qscintilla.override { withQt5 = true; });
+
+in mkDerivation rec {
+  version = "3.10.1";
   name = "sqlitebrowser-${version}";
 
   src = fetchFromGitHub {
     repo   = "sqlitebrowser";
     owner  = "sqlitebrowser";
     rev    = "v${version}";
-    sha256 = "1fwr7p4b6glc3s0a06i7cg8l9p1mrcm4vyhyf2wi89cyg22rrf5c";
+    sha256 = "1brzam8yv6sbdmbqsp7vglhd6wlx49g2ap8llr271zrkld4k3kar";
   };
 
-  buildInputs = [ qtbase qttools sqlite ];
-  nativeBuildInputs = [ cmake ];
+  buildInputs = [ qtbase qscintillaLib sqlite ];
+
+  nativeBuildInputs = [ cmake antlr qttools ];
+
+  enableParallelBuilding = true;
 
   cmakeFlags = [ "-DUSE_QT5=TRUE" ];
 
   # A regression was introduced in CMakeLists.txt on v3.9.x
   # See https://github.com/sqlitebrowser/sqlitebrowser/issues/832 and issues/755
   postPatch = ''
-    substituteInPlace CMakeLists.txt --replace 'project("DB Browser for SQLite")' 'project(sqlitebrowser)'
+    substituteInPlace CMakeLists.txt \
+      --replace 'project("DB Browser for SQLite")' 'project(sqlitebrowser)' \
```

Comment:
That substitution is not needed anymore: CMakeLists.txt was fixed upstream for 3.10.x
You can also remove the 2 lines of comments above regarding this past issue.

Reply:
Done (and thanks!)